candle_cruncher.py

```diff
@@ -82,11 +82,11 @@
 
 # Get fragrance dosing
 if option == 'Christmas Day':
-    manual_weight = 0.08125*(volume*density)
+    manual_weight = 0.08125*(manual_vol*density)
```

Comment:
This works (I think) to make the "manual volume" always match the input volume. However, when people are trying get volume from a manual length measurement. This overrides that. In other words, if I select a certain length in the app it doesn't cause any change with this modification. We want to synchronize the numbers between sections, so I think we will need to do something else.